frontend-web/webclient/app/Applications/ApplicationsRow.tsx

```diff
@@ -0,0 +1,168 @@
+import {injectStyle} from "@/Unstyled";
+import {ApplicationGroup} from "./api";
+import {AppCard, ApplicationCardType} from "./Card";
+import React, {useEffect, useState} from "react";
+import {Absolute, Flex, Grid, Icon, Link, Relative} from "@/ui-components";
+import * as Pages from "./Pages";
+
+const ApplicationRowContainerClass = injectStyle("tag-grid-bottom-box", k => `
+    ${k} {
+        padding: 15px 10px 15px 10px;
+        margin: 0 -10px;
+        overflow-x: auto;
+    }
+`);
+
+interface ApplicationRowProps {
+    items: ApplicationGroup[];
+    type: ApplicationCardType;
+    refreshId: number;
+    scrolling: boolean;
+}
+
+const SCROLL_SPEED = 156 * 4;
```

Comment:
This is fine as is, but a comment as to why it is multiplied by 4 would be nice.